Plugin/CsrfValidatorSkip.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ *
+ *        c.                                            c.  :.
+ *        E1                                            E.  !)
+ * ::.    E1    !3.        ,5"`'\.;F'`"t.   i.   cF'`'=.E.  !7''`   ;7""   '""!.    ;7"'"!.   ;7'`'*=
+ * ::.    E1    !3.        t.    !)     t   i.  tL      t.  !)     !)     ,,...;)  :1     I.  !t.,
+ * ::.    E1    !3.        t.    !)     t   i.  E.      E.  !)     !L    t'    :1  :1     !)    ``"1.
+ * ::.    E1    !3.        t.    !)     t   i.  '1.,  ,ct.  !1,    !L    1.  ,;31  :1     !) -..   ;7
+ * '      E1    `'         `            `   `     ``'`  `    `'``  `      `''`  `   `     `    ``'`
+ *        E7
+ *
+ * Midtrans Snap Magento 2 Module
+ *
+ * Copyright (c) 2020 Midtrans PT.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
```

Comment:
No need to include copyright, to encourage people to contribute. No need to state MIT license, as long as the root folder have the License file. We can remove this.

You may however add `author` and your name/contact on header comment of important files if you want.

Reply:
done, removed